src/ARCH2018_RE.jl

```diff
@@ -1,6 +1,4 @@
 # this is a wrapper module for the benchmarks
 module ARCH2018_RE
-    @static if VERSION >= v"0.7"
-        using SparseArrays, LinearAlgebra
```

Comment:
What is the purpose of this file?

Reply:
This is an empty module file but with the commands with `PkgBenchmark` works:

```julia
using PkgBenchmark, Reachability
results = benchmarkpkg("ARCH2018_RE") # needs a module
export_markdown("results.md", results)
```

That said, at the time i didn't know that one is able to run a benchmark suite without the need of a proper package (as we know do in ReachabilityBenchmarks! you know like using [run](https://github.com/JuliaReach/ReachabilityBenchmarks/blob/master/models/ARCH/AFF/SpaceStation/iss_benchmark.jl#L17)) -- i haven't checked if this can be done with those tools at the v0.6 state.

.. If we could use it that way it would make things cleaner for people usng this RE. For example, if you've cloned the package in you `~/home/`, and run the install script, you end up with 2 copies one in `~/home/ARCH2018_RE` and one in `~/home/.julia/v0.6/ARCH2018_RE`. If you want to test something in the model, the script that has to be modified is the on in the copy located in `.julia/v0.6/ARCH2018_RE`! The files in `~/home/ARCH2018_RE` are ignored if i call `benchmarkpkg`.

Reply:
Okay, then just don't clone this repository by hand :smile: